text/0000-yarn-update-transitive.md

```diff
@@ -0,0 +1,55 @@
+- Start Date: 2017-03-18
+- RFC PR: (leave this empty)
+- Yarn Issue: (leave this empty)
+
+# Summary
+
+This RFC proposes the addition of a "update-transitive" command to the Yarn CLI.

```

Comment:
I think it should be `upgrade-transitive` to match `upgrade` and `upgrade-interactive`, otherwise it looks like it goes with `yarn self-update`

Reply:
Yep, this makes sense, thanks!